shell/platform/windows/win32_dpi_helper.cc

```diff
@@ -43,6 +43,20 @@ Win32DpiHelper::~Win32DpiHelper() {
   }
 }
 
+void Win32DpiHelper::SetDpiAwerenessAllVersions() {
+  if (permonitorv2_supported_) {
+    SetProcessDpiAwarenessContext(DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2);
+  } else {
+    std::cerr << "Per Monitor V2 DPI awareness is not supported on this "
```

Comment:
Why is this logged? It's not an error case, unless I'm not understanding the PR.

Reply:
I saw it as more of a fringe case, but it's true it's not an error. Makes sense to remove it.

Reply:
Done